bliss/catalog.py

```diff
@@ -126,7 +126,7 @@ class TileCatalog(BaseTileCatalog):
     galaxy_params_index = {k: i for i, k in enumerate(galaxy_params)}
 
     def __init__(self, d: Dict[str, Tensor]):
-        self.max_sources = d["locs"].shape[3]
+        assert "locs" in d
```

Comment:
Could we also assert that locs is a 5d tensor here?

Reply:
have fixed